frontend/src/components/AccountReceiveTableWarning.vue

```diff
@@ -11,24 +11,21 @@
     </q-card-section>
 
     <q-card-section>
-      <p>
-        You are withdrawing to <span class="code">{{ destinationAddress }}</span
-        >, which has the following warnings:
-      </p>
+      <i18n path="AccountRTW.withdrawal-warning" tag="p">
```

Comment:
I think it'd be better if the JSON keys are the full file name instead of acronyms. It's more verbose, but more clear and avoids conflicts where files might get abbreviated the same way. For example, how does this naming scheme work with `AccountSetup` and `AccountSend`?